eks-worker-al2.json

```diff
@@ -58,6 +60,8 @@
       "tags": {
           "created": "{{timestamp}}",
           "docker_version": "{{ user `docker_version`}}",
+          "source_ami_id": "{{ user `source_ami_id`}}",
+          "source_ami_filter_name": "{{ user `source_ami_filter_name`}}", 
```

Comment:
Can you remove this tag? It is just a query filter, and `source_ami_id` should be enough to debug any issues

Reply:
Done